lib/0001_retrieve-secrets.sh

```diff
@@ -68,12 +72,14 @@ def generate_vcap_creds_hash
     creds.has_key? key.to_s
   end
 
+  @creds_map.reject! { |key, _| creds[key.to_s].empty? }
+
   if !valid_creds
     STDERR.puts("malformed '#{@service_label}' credentials in VCAP_SERVICES")
     exit(1)
   end
 
-  Hash[ @creds_map.map { |key, env_var_name| [ env_var_name, creds[key.to_s] ] } ]
+  Hash[ @creds_map.map { |key, env_var_name| [ env_var_name, creds[key.to_s].to_s ] } ]
```

Comment:
What are the implications of this `to_s`? It's just converting the `CONJUR_VERSION`? Does this end up mattering somewhere?

Reply:
it's converting the CONJUR_VERSION to a string - it throws an error without it (I didn't identify exactly where), and doesn't affect anything downstream with it